prow/config.yaml

```diff
@@ -574,6 +574,7 @@ presubmits:
     context: pull-ingress-gce-test
     branches:
     - master
+    - mci-dev
```

Comment:
you might want to unrestrict the branches to all or consider switching to blacklisting branches instead.

Reply:
yeah, as I've mentioned, get rid of the `branches:` field will achieve the same behavior